plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/Transform.cpp

```diff
@@ -105,10 +104,11 @@ MStatus Transform::initialise()
   try
   {
     setNodeType(kTypeName);
+    inheritAttributesFrom("AL_usdmaya_Scope");
 
     addFrame("USD Prim Information");
-    m_primPath = addStringAttr("primPath", "pp", kReadable | kWritable | kStorable | kConnectable | kAffectsWorldSpace, true);
-    m_inStageData = addDataAttr("inStageData", "isd", MayaUsdStageData::mayaTypeId, kWritable | kStorable | kConnectable | kHidden | kAffectsWorldSpace);
+    addInheritedAttr("primPath");
+    addInheritedAttr("inStageData");
```

Comment:
FYI I am spending some time today trying to get it built and see if the tests pass.  This is a line you'll have problems with.  Paul removed this function and replaced it with something slightly different.  See commit 476826f1